zxinglight/_zxinglight.cpp

```diff
@@ -67,24 +67,19 @@ static vector<string> *_zxing_read_codes(
 
         Ref<BinaryBitmap> bitmap(new BinaryBitmap(binarizer));
 
-        vector<Ref<Result> > results;
+        vector<Ref<Result> > *results;
```

Comment:
I'm not sure why I made this a pointer. There was a reason. Maybe it can go back to not being a pointer.

Reply:
Oh right, it's because the code below uses _zxing_read_codes() == NULL to signal an error